automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala

```diff
@@ -77,6 +77,21 @@ class WorkspaceSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures
 
       }
 
+      "and who owns the project" - {
+        "should see the Project Cost section of the summary page" in withWebDriver { implicit driver =>
+          val user = UserPool.chooseProjectOwner
+          implicit val authToken: AuthToken = authTokenOwner
+          val testName = "WorkspaceSpec_projectOwnerAccess_projectCost"
+          withWorkspace(billingProject, testName, Set.empty, List.empty) { workspaceName =>
+            withSignIn(user) { listPage =>
+              val workspacePage = listPage.enterWorkspace(billingProject, workspaceName)
+              workspacePage.hasGoogleBillingLink shouldBe true
+              workspacePage.hasStorageCostEstimate shouldBe true
+            }
+          }(authTokenOwner)
```

Comment:
Are these necessary?

Reply:
Doesn't look like it, but it makes me wonder how they got there. First thoughts are:
1. Copypasta
1. Refactoring
If 1 then they're probably safe to remove. If 2 then I worry that the intent of the test may have changed. Maybe the implicit token in this scope was changed from something else. If that was done intentionally, I would expect the explicit parameter to have been removed at the same time.

Either way, it's an accident and I think it would be prudent to double-check the test to make sure it's still exercising the intended behavior.